ThesesFR.js

```diff
@@ -0,0 +1,165 @@
+{
+	"translatorID": "3f73f0aa-f91c-4192-b0d5-907312876cb9",
+	"label": "ThesesFR",
+	"creator": "TFU",
+	"target": "^https?://(www\\.)?(theses\\.fr)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-04-23 11:16:01"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	theses.fr
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+ 
+	if (url.indexOf("?q=") != -1) {
+		return "multiple";
+	} else if (url.indexOf("?q=") == -1) {
```

Comment:
ditto

Reply:
also, take advantage of `checkSearchResults` below and use that here to avoid false positives.

Reply:
Hi,
Theses.fr is Ok : link works : http://www.theses.fr/s197772
Thanks a lot for your reviewing : i'll make the changes asap!